src/Abstractions/TaskActivityBase.cs

```diff
@@ -94,6 +94,7 @@ public abstract class TaskActivityBase<TInput, TOutput> : ITaskActivity
     Exception DefaultNotImplementedException()
     {
         return new NotImplementedException(
-            $"{this.GetType().Name} needs to override {nameof(this.OnRun)} or {nameof(this.OnRunAsync)} with an implementation.");
+            $"{this.GetType().Name} needs to override {nameof(this.OnRun)} or {nameof(this.OnRunAsync)} with an"
+            + " implementation.");
```

Comment:
Part of me feels like we're following this line length guideline a bit too religiously. I think we should consider it more of a guide to help keep the code readable. Changes like these, however, where we introduce a newline for a single word seem to make the code less readable, IMO. Thoughts?

Reply:
Yeah that is fair - I will undo those changes.

Reply:
I undid some of them. If the last word/statement crosses 120 I will leave it be. If an entirely new word starts past 120, then I move everything 120 and up to the new line.